src/main/java/sx/blah/discord/handle/obj/IUser.java

```diff
@@ -153,4 +153,17 @@
 	 * @return True if muted, false if otherwise.
 	 */
 	boolean isMuted(IGuild guild);
+
+	/**
+	 * Gets whether this user is deafened locally.
+	 *
+	 * @return True if deafened, false if otherwise.
+     */
+	boolean isDeafLocally();
```

Comment:
I think that the JavaDoc should explain that locally = self because it may be confusing to a user as to what the difference between being locally deaf and normally deaf